include/RMGVertexConfinement.hh

```diff
@@ -179,6 +306,7 @@ class RMGVertexConfinement : public RMGVVertexGenerator {
     bool fOnSurface = false;
     bool fForceContainmentCheck = false;
     bool fLastSolidExcluded = false;
+    int fMaxNumIntersections = -1;
```

Comment:
Like for the macro command, let's use a more precise name. Like adding `GSS` in the name